packages/commonwealth/client/scripts/state/api/launchPad/buyToken.ts

```diff
@@ -28,9 +33,55 @@ const buyToken = async ({
   return await launchPad.buyToken(amountEth, walletAddress, `${ethChainId}`);
 };
 
+export const handleQuerySuccess = async (
```

Comment:
its used on sell and both so lets move it to separate file

Reply:
hm, i moved it initially, but thought it might become another `cache.ts` similar to threads. lmk if you still think it'd be good to move it.